server/src/test-utils/environments/mock-entity-mikro-environment.ts

```diff
@@ -1,43 +1,22 @@
-import { mocks } from '..'
-import { ClubEntity } from '../../shared/infra/db/entities/club.entity'
-import { EventEntity } from '../../shared/infra/db/entities/event.entity'
-import { TagEntity } from '../../shared/infra/db/entities/tags/tag.entity'
+//import { mocks } from '..'
```

Comment:
whats the purpose of this mock/why was it essentially removed?

Reply:
`mocks` is a module containing a bunch of methods that create mock objects; we then use the objects in testing

It was used to generate entities that populate the clubs/events DBs, but those aren't needed anymore. If you need to create some mock users for testing, you would create a "constructor" in the `mocks` module and import it here.

Reply:
ahh sorry, I didn't notice that the entities removed were things that didn't need to be part of the `auth` repo, I thought we'd removed all of those when we cleaned up the repository.

Reply:
Actually looking more closely now I think you can delete this whole file. If your tests don't need global mock entities then you can simply use the `MikroTestEnvironment` instead.